tests/ckeditor.js

```diff
@@ -104,14 +104,15 @@ describe( 'CKEditor Component', () => {
 				expect( vm.value ).to.equal( '' );
 			} );
 
-			it( 'should set the initial data', done => {
-				const setDataStub = sandbox.stub( MockEditor.prototype, 'setData' );
-				const { wrapper } = createComponent( {
+			it( 'should set the initial data by using innerHTML, not by "setData()"', done => {
+				const { wrapper, vm } = createComponent( {
 					value: 'foo'
 				} );
 
 				Vue.nextTick( () => {
-					sinon.assert.calledWithExactly( setDataStub, 'foo' );
+
+					expect( vm.$el.innerHTML ).to.equal( 'foo' );
+					expect( vm.instance.setDataCounter).to.equal(0);
```

Comment:
Also: use `sinon.spy( vm.instance, 'setData' )` and `sinon.assert.notCalled( spy )` instead of the counter. Don't spread the logic across the files when it's unnecessary.

Reply:
`sinon.spy( vm.instance, 'setData' )` was problematic in this case. You can not do it before `Vue.nextTick`, because `vm.instance` was `null`, and you can not do it after `Vue.nextTick` because the editor is already initialized then. Also, note that this counter does not change the logic of the `EditorMock`, it just keep additional information.